src/features/rewards/amount/spec.tsx

```diff
@@ -6,7 +6,7 @@ import Amount from './index'
 import { TestThemeProvider } from '../../../theme'
 
 describe('Amount tests', () => {
-  const baseComponent = (props?: object) => <TestThemeProvider><Amount id='amount' amount={1} converted={0.4} onSelect={() => {}} {...props}  /></TestThemeProvider>
+  const baseComponent = (props?: object) => <TestThemeProvider><Amount id='amount' amount={'1'} converted={'0.4'} onSelect={() => {}} {...props} /></TestThemeProvider>
```

Comment:
I also made changes here for the testing. Types should be clear now :)

Reply:
nice find